docs/dev/test-configuration-options.md

```diff
@@ -714,45 +714,19 @@ Available visibility modes are:
 "public": "team"
 ```
 
----
-### `tunnelName`
-<p><small>| STRING | </small></p>
-
-Specify a [Sauce Connect](/secure-connections/sauce-connect) tunnel to establish connectivity with Sauce Labs for your test. Tunnels allow you to test an app that is behind a firewall or on your local machine by providing a secure connection to the Sauce Labs platform.
-
-See [Basic Sauce Connect Proxy Setup](/secure-connections/sauce-connect/setup-configuration/basic-setup) for more information.
-
-```java
-"tunnelName": "MyTunnel01"
-```
-
----
-### `tunnelOwner`  
-<p><small>| STRING |</small></p>
-
-If the [tunnelName](#tunnelname) you've specified to establish connectivity with Sauce Labs for your test is a shared tunnel, and you are _not_ the user who created the tunnel, you must identify the Sauce Labs user who did create the tunnel in order to use it for your test.
-
-See [Using Tunnel Names](/secure-connections/sauce-connect/setup-configuration/basic-setup/#using-tunnel-names) for more information.
-
-
-```java
-"tunnelName": "MyTeamSharedTunnel"
-"tunnelOwner": "<username of tunnel originator>"
-```
```

Comment:
@mdsauce Since `tunnelName` and `tunnelOwner` fall under **Desktop _and_ Mobile Capabilities**, was thinking we should leave them here and apply the RDC Only badge ([like this](https://docs.saucelabs.com/dev/test-configuration-options/#allowtouchidenroll)).

cc: @sweeneyskirt-sl @waggledans

Reply:
Thanks, @kimsauce. The directive is to remove the new nomenclature altogether until it works for all environments. The old caps are still functional, so it is less confusing to our customers to just stick with the one set of caps that works everywhere.